app/src/main/res/xml/grayscale_icon_map.xml

```diff
@@ -458,6 +460,7 @@
     <icon drawable="@drawable/grab" package="com.grabtaxi.passenger" name="Grab" />
     <icon drawable="@drawable/grapheneos_apps" package="app.grapheneos.apps" name="Apps" />
     <icon drawable="@drawable/grapheneos_auditor" package="app.attestation.auditor" name="Auditor" />
+    <icon drawable="@drawable/grapheneos_pdf" package="app.grapheneos.pdfviewer" name="PDF Viewer" />
     <icon drawable="@drawable/grapheneos_pdf" package="app.grapheneos.pdfviewer.play" name="Secure PDF Viewer" />
```

Comment:
This must be rename to `PDF Viewer` only.
![image](https://user-images.githubusercontent.com/40175769/201284812-f33463ea-8a9b-4a8b-af3e-3ef7ef0cadbb.png)

Reply:
I just added what you added, if there is an issue, please send a new PR to fix.

Reply:
#820